apps/test/unit/templates/instructions/RubricFieldTest.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import {UnwrappedRubricField as RubricField} from '@cdo/apps/templates/instructions/RubricField';
+
+const DEFAULT_PROPS = {
+  showFeedbackInputAreas: true,
+  rubricLevel: 'exceeds',
+  rubricValue: 'exceeded expectations',
+  disabledMode: false,
+  onChange: () => {},
+  currentlyChecked: false
+};
+
+describe('RubricField', () => {
+  it('renders with the correct values', () => {
+    const wrapper = shallow(<RubricField {...DEFAULT_PROPS} />);
+
+    // Radio Button
+    const confirmCheckedRadioButton = wrapper
+      .find('CheckedRadioButton')
+      .first();
+    expect(confirmCheckedRadioButton.props().checked).to.equal(false);
+
+    // Details
+    const confirmDetails = wrapper.find('details').first();
+    expect(confirmDetails).to.contain('Exceeds');
```

Comment:
might cause a break when chai-enzyme is removed (you can confirm by replacing 'configuredChai' with 'reconfiguredChai'
If needed, you can replace with:
expect(dialog.find('Exceeds')).to.be.empty();
Examples: https://github.com/code-dot-org/code-dot-org/pull/27553/files